docs/_dg/UseCases.md

```diff
@@ -28,9 +28,9 @@ Extensions:
 
 #### Use Case 2: Update Item in Inventory
 
-Preconditions: Item to edit exists in inventory.
+Preconditions: [[ item:Item ]] to edit exists in inventory.
 
-MSS:
+[[ MSS ]]:
```

Comment:
Like OS, this is an invalid key as the glossary entry name is "MSS (Main Success Scenario)". However, I'm going to change this to "Main Success Scenario (MSS)" in another PR, so let's fix it the right way.

```suggestion
[[ main-success-scenario-mss:MSS ]]:
```